torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -27,6 +27,24 @@ namespace cuda {
 
 int FusionExecutor::fusion_id_counter_ = 0; // NOLINT
 
+namespace {
+
+static const char* defineIndexMode(KernelIndexMode index_mode) {
+  switch (index_mode) {
+    case KernelIndexMode::INT32:
+      return "#define INDEX_TYPE int\n";
```

Comment:
nit: we already have typedefs for some integer types. Why not another typedef?

Reply:
I changed it to typedef and I agree that it'd be easier to maintain. It was just that using typedef will require INDEX_TYPE to be defined after the typedefs, so I moved the defs around a little bit.